tests/cases/tm2tm/scripts/test-service

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+: <<'END_COMMENT'
+* relay-interval = 20s
+* src-relay-optimize-interval = 30s
+* src-relay-optimize-count = 3
+* dst-relay-optimize-interval = 30s
+* dst-relay-optimize-count = 3
+
+- relay service [packets = 0, time = 0] -> skip
+    - sleep 4
+    - transfer x 1
+    - packets = 1, ack = 0
+    - sleep 16
+- relay service [packets = 1, time = 20] -> skip
+    - sleep 4
+    - transfer x 1
+    - packets = 2, ack = 0
+    - sleep 16
+- relay service [packets = 2, time = 40] -> exec (time)
+    - sleep 4
+    - packets = 0, ack = 2
+    - transfer x 3
+    - packets = 3, ack = 2
+    - sleep 16
+- relay service [packets = 3, time = 20] -> exec (count)
+    - sleep 4
+    - packets = 0, ack = 3
```

Comment:
The number of acks is wrong. `ack = 5` is correct, right?

Reply:
FIxed test case comments
https://github.com/hyperledger-labs/yui-relayer/pull/108/commits/72e4d888541253c9f6e0a953d9883bd428807eda